### ISSUE TITLE
A.1.1 ISO Metadata Application Profile

### ISSUE THREAD

**Issue by jensscheerlinck** [inspire-eu-validation/ats-discovery-service#9]
_This issue has been extracted from the issue list on:https://ies-svn.jrc.ec.europa.eu/issues/2685_
# Comment

The Test Method requires testing OGC standard functionality, which instead shall be delegated to the OGC Tests.
The Test Method shall instead just inspect the capabilities document and this is not mentioned.

**Comment by jensscheerlinck** [comment]
# Proposed Resolution

This test case should be considered as an abstraction of the other test cases within the Discovery Service ATS (which cover in detail the Capabilities Document).

**Comment by PeterParslow** [comment]
I do not agree with the proposed resolution. This test case is the basic one which should be conducted before proceeding to the INSPIRE specific test cases.
I do agree with the initial comment, testing this case should ideally be by http://cite.opengeospatial.org/teamengine/about/csw/2.0.2/site/ - except that that suite requires loading test data. Until OGC provide a set of tests which can be run without test data, we will have quite a 'hole' in our validation.
See also https://github.com/inspire-eu-validation/ats-cross-cutting/issues/7

**Comment by PeterParslow** [comment]
See #28, which proposes working with OGC on this.
Note also that this is an ISO AP specific suite, not the generic CSW one referenced above.

**Comment by cportele** [comment]
As I commented in https://github.com/inspire-eu-validation/ats-view-wms/issues/127

An option could be to have a manual test case to verify that the CSW software used is certified as compliant by OGC. That still does not fill the gap regarding an CSW ISO-AP CITE test, if there is none.

**Comment by jensscheerlinck** [comment]
Added the label feedback:for-discussion based on the comment from @cportele

**Comment by jensscheerlinck** [comment]
Based on the discussion held during the MIWP-5 meeting:

On ATS level, conformance to a third-party specification or standard will be expressed as a reference to the relevant conformance class(es). Deciding which ETSs to use for this will be decided during the implementation of the ETSs. If no third-party Test Suite is available, we could rely on self declaration (as we will not develop our own tests for third-party specifications or standards).

Therefore I believe the current proposed resolution would be to add the correct OGC ISO AP specification as a reference and perhaps to update the note stating that we will not develop our own ETS.
